src/libraries/Microsoft.VisualBasic.Core/tests/ConversionsTests.cs

```diff
@@ -2749,28 +2697,6 @@ public static object DoubleEnum
             }
         }
 
-        private static object s_boolEnum;
```

Comment:
It would be nice to keep some tests that verify that the bool backed enums can be still loaded by the type loader.

Reply:
Would the `src/tests/JIT/Intrinsics/TypeIntrinsics.cs` IsEnum test that's removed here be a good choice for that?

Reply:
Yes, I agree that we should keep `TypeIntrinsics.cs` test.